content/en/docs/tasks/configure-pod-container/resize-container-resources.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Resize CPU and Memory Resources assigned to Containers of a Pod
+content_template: templates/task
+weight: 20
+---
+
+{{% capture overview %}}
```

Comment:
@vinaykul it looks like you are using a page template from a few years ago. Check other pages within `content/en/docs/tasks/` for an example of what a task page looks like now.

Reply:
Thanks for the feedback Tim. Yes this feature has gone through a few redesigns and I had written up the documentation when the earlier design was near implemented and we decided that it needed to be reworked. I'll update to use the latest template in the next milestone.